activerecord/test/cases/finder_test.rb

```diff
@@ -291,15 +291,15 @@ def test_exists_returns_false_with_false_arg
 
   def test_exists_with_loaded_relation
     topics = Topic.all.load
-    assert_no_queries do
+    assert_queries_count(1) do
```

Comment:
Not purely necessary, but you could use `assert_queries_match` to ensure it's the SELECT COUNT query, and not another unexpected one. But the call is small enough that it's probably not worth it.